### PR TITLE
fix: [CDS-51827]: Fixed Failing SchedulePanel.test.tsx

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cd

**Comment by pankajsingh-harness** [comment]
retrigger titlecheck

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cd

**Comment by pankajsingh-harness** [comment]
retrigger cypress-pipeline

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cd

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cv

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cv

**Comment by pankajsingh-harness** [comment]
retrigger cypress-pipeline

**Review by ishantmehtaharness — DISMISSED** [review]

File: src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx [hunk 1]
```diff
@@ -191,7 +191,10 @@ describe('SchedulePanel Triggers tests', () => {
 
       const options = document.querySelectorAll('[class*="bp3-menu"] li')
       fireEvent.click(options[3])
-      await waitFor(() => expect(queryByText(container, '0 0/4 * * *')).not.toBeNull())
+
+      await waitFor(() =>
+        expect(document.querySelector('p[data-testid="expression"]')?.textContent).toBe('0 0/4 * * *')
```
[the remark this answers is not in the document]
Comment (pankajsingh-harness) [reply src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx]: Done

**Review by sunnykesh-harness — DISMISSED** [review]

**Review by satyam-harness — DISMISSED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Looks good

File: src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx [hunk 2]
```diff
@@ -191,7 +191,8 @@ describe('SchedulePanel Triggers tests', () => {
 
       const options = document.querySelectorAll('[class*="bp3-menu"] li')
       fireEvent.click(options[3])
-      await waitFor(() => expect(queryByText(container, '0 0/4 * * *')).not.toBeNull())
+
+      await waitFor(() => expect(queryByTestId(container, 'expression')?.textContent).toBe('0 0/4 * * *'))
```
Comment (AshwinBhatkal-Harness) [inline src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx:195]: nit: queryByTestId would be returned by the render of the component. We don't need a specific import just to specify the container again

File: src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx [hunk 3]
```diff
@@ -266,9 +268,10 @@ describe('SchedulePanel Triggers tests', () => {
       const amPmOptions = document.querySelectorAll('[class*="bp3-menu"] li')
       fireEvent.click(amPmOptions[1])
       // tests pm adds 12 hours
-      await waitFor(() => expect(queryByText(container, '3 12 * * *')).not.toBeNull())
+
+      await waitFor(() => expect(queryByTestId(container, 'expression')?.textContent).toBe('3 12 * * *'))
     })
-    // eslint-disable-next-line jest/no-disabled-tests
+    // eslint-disable-next-line jest/no-disabled-tests, jest/expect-expect
```
Comment (AshwinBhatkal-Harness) [inline src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx:274]: Why have added this comment? We are expecting at the end of the test

Comment (pankajsingh-harness) [reply src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx:274]: Done

File: src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx (hunk 2 above)
Comment (pankajsingh-harness) [reply src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx:195]: Done

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by Eric761 — APPROVED** [review]

**Pull request merged**